minitests/timing/Makefile

```diff
@@ -27,7 +27,6 @@ build/$(2)_$(1)/timing_$(2)_$(1).json5: build/.touch runme.tcl $(3)
 build/timing_$(2)_$(1).xlsx: build/$(2)_$(1)/timing_$(2)_$(1).json create_timing_worksheet_db.py
 	python3 create_timing_worksheet_db.py \
 		--timing_json build/$(2)_$(1)/timing_$(2)_$(1).json \
-		--db_root ${XRAY_DATABASE_DIR}/${XRAY_DATABASE} \
```

Comment:
No, add both the `--db_root` argument and `--part` argument

Reply:
Fixed